spec/team_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper.rb'
+
+RSpec.describe Team do
+  it "can initialize" do
```

Comment:
thank you for going above and beyond to do the spec tests for game, team, game_teams, and stat_tracker. I knew someone was going to have to do that eventually!

Reply:
Yeah thanks Chee. Your a legend man.